paper.md

```diff
@@ -24,20 +24,20 @@ bibliography: paper.bib
 
 # Summary
 
-Developments in the field of mobile robotics and autonomous driving have resulted in robots and vehicles in retail stores, hospitals, warehouses, on the roads, and on sidewalks.
+Developments in the field of mobile robotics and autonomous driving have resulted in the use of robots and vehicles in retail stores, hospitals, warehouses, on the roads, and on sidewalks.
 These deployed areas are both dynamic and frequently massive in scale.
-The average size of a Walmart store is over 16,000 $m^{2}$ [CITATION NEEDED] and a single square block in Chicago is over 21,000 $m^{2}$ [CITATION NEEDED].
+The average size of a Walmart store is over 16,000 $m^{2}$ [@Walmart] and a single square block in Chicago is over 21,000 $m^{2}$ [CITATION NEEDED].
```

Comment:
Not sure what to put under the Chicago related citation.

Reply:
I'll figure it out, thanks!